version.txt

```diff
@@ -1 +1 @@
-v6.1.0
+v6.2.0
```

Comment:
If the previous actions are deleted, it's better to bump the major, i.e. `v7.0.0`